lighthouse-core/computed/page-dependency-graph.js

```diff
@@ -144,16 +144,19 @@ class PageDependencyGraph {
    */
   static linkNetworkNodes(rootNode, networkNodeOutput) {
     networkNodeOutput.nodes.forEach(node => {
+      const directInitiatorRequest = node.record.initiatorRequest || rootNode.record;
+      const directInitiatorNode =
```

Comment:
the `|| rootNode` should be unnecessary here because of the `||` above, so this is just for ts, right? or maybe I'm missing something :)

Reply:
Agreed - I would be surprised if the `get` call ever fails for the root node. I went this way since I think it's the least verbose option to keep the compiler happy, but I can change it to a type assertion if you prefer

Reply:
oh no no I agree this is the best way, just double checking my understanding 👍